app/src/main/res/layout/toolbar_main.xml

```diff
@@ -1,97 +1,90 @@
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools">
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/toolbar_root"
+    android:layout_width="match_parent"
+    android:layout_height="60dp">
 
-    <data>
+    <ImageView
+        android:id="@+id/toolbar_back"
+        android:layout_width="@dimen/button_height_32dp"
+        android:layout_height="@dimen/button_height_32dp"
+        android:contentDescription="@string/image_desc"
+        android:paddingStart="@dimen/margin_new_tiny"
+        android:paddingEnd="@dimen/margin_new_tiny"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@drawable/ic_back_red" />
 
-    </data>
+    <TextView
+        android:id="@+id/toolbar_title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textColor="@color/white_light"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintLeft_toRightOf="@+id/toolbar_back"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="Single Title" />
 
-    <androidx.constraintlayout.widget.ConstraintLayout
-        android:id="@+id/toolbar_root"
-        android:layout_width="match_parent"
-        android:layout_height="60dp">
+    <TextView
+        android:id="@+id/toolbar_main_title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textColor="@color/white_light"
+        app:layout_constraintBottom_toTopOf="@+id/toolbar_sub_title"
+        app:layout_constraintLeft_toRightOf="@+id/toolbar_back"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_chainStyle="packed"
+        tools:text="Main title" />
 
-        <ImageView
-            android:id="@+id/toolbar_back"
-            android:layout_width="@dimen/button_height_32dp"
-            android:layout_height="@dimen/button_height_32dp"
-            android:contentDescription="@string/image_desc"
-            android:paddingStart="@dimen/margin_new_tiny"
-            android:paddingEnd="@dimen/margin_new_tiny"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintLeft_toLeftOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
-            app:srcCompat="@drawable/ic_back_red" />
+    <TextView
+        android:id="@+id/toolbar_sub_title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textColor="@color/white_light"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintLeft_toRightOf="@+id/toolbar_back"
```

Comment:
use the `Start` and `End` to replace `Left` and `Right`, because in some regions, the screen starts from right to end.